json-schema/json-schema-generic/src/main/scala-3/endpoints4s/generic/JsonSchemas.scala

```diff
@@ -119,7 +119,7 @@ trait JsonSchemas extends algebra.JsonSchemas:
     * @param  defaultValues  List of the possible default values of the record fields (e.g.,
     *                        `(Some(42), None)`)
     */
-  private[generic] inline def summonRecord[
+  inline def summonRecord[
```

Comment:
Why were these changes necessary? I wish these methods could be kept `private`.

Reply:
Scala 3 compilation was complaining in Fetch JsonSchema tests about failing to summon these methods. I think other interpreters that have these tests are not compiled for Scala 3 yet.

Reply:
Indeed, this is due to https://github.com/lampepfl/dotty/issues/11706 but that issue does not seem to have been prioritized.

Reply:
Let’s come back to this Scala 3 issue when we become ready to publish Scala 3 artifacts…